package.json

```diff
@@ -16,8 +16,8 @@
 		"build": "yarn install-if-deps-outdated && yarn clean && yarn build-php && yarn build-packages && yarn build-jetpack && yarn build-jetpack-backup",
 		"build-concurrently": "yarn install-if-deps-outdated && yarn clean && yarn concurrently 'yarn build-php' 'yarn build-packages' 'yarn build-jetpack-concurrently' 'yarn build-jetpack-backup-concurrently'",
 		"build-jetpack": "cd projects/plugins/jetpack && yarn build",
-		"build-jetpack-backup": "cd projects/plugins/jetpack-backup && yarn build",
-		"build-jetpack-backup-concurrently": "cd projects/plugins/jetpack-backup && yarn build-concurrently",
+		"build-jetpack-backup": "cd projects/plugins/backup && yarn build",
```

Comment:
100% absolutely not a blocker (let's merge this!),  but is there a need for this vs using `jetpack build plugins/backup` command?

Ideally, we can use the CLI and not require adding these steps for every project.

Reply:
I agree, it might be excessive, although I'm a bit behind on understanding of the modern Jetpack devops 🙂 
When I recommended @samiff to use it [in another PR](https://github.com/Automattic/jetpack/pull/19332#issuecomment-821680665), I took the `build-jetpack` commands as an example.
If they're no longer needed, we could remove them in #19716, which will be merged after this one.